package.json

```diff
@@ -16,10 +16,14 @@
     "node": ">=4"
   },
   "scripts": {
-    "test": "xo && ava"
+    "lint": "tslint --project .",
+    "build": "tsc",
+    "test": "npm run lint && npm run build && ava dist/tests",
+    "prepublish": "npm run build && del dist/tests"
   },
+  "main": "dist",
```

Comment:
Add `"typings": "dist/index.d.ts"` here as well. Not sure if needed in the `main` field  though, but I always point to the `index.js` file explicitely.

```json
"main": "dist/index.js",
"typings": "dist/index.d.ts",
```